test/test_seqr_loader_dry.py

```diff
@@ -188,7 +188,9 @@ def mock_create_new_analysis(*args, **kwargs) -> int:
 
     mocker.patch('pathlib.Path.open', selective_mock_open)
     # functions like get_intervals checks file existence
-    mocker.patch('cloudpathlib.cloudpath.CloudPath.exists', mock_exists)
+    mocker.patch('cpg_workflows.workflow.list_all_parent_dirs', lambda *args: {})
+    mocker.patch('cpg_workflows.workflow.list_of_all_dir_contents', lambda *args: {})
+    mocker.patch('cpg_workflows.workflow.exists_on_pre_collected', lambda *args: False)
```

Comment:
Lambda's return should be `None`.

Reply:
good spot

Reply:
Somehow despite being a copy-paste from the other test case, I only got it wrong here 🙃